2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -70,21 +70,33 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Word Search
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/word-search/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given a 2D board and a word, find if the word exists in the grid.
```

Comment:
OH HEY ITS BOGGLE <3

You should cover this for this session, but [be aware that we've covered an extremely similar problem recently for the Graphs session](https://github.com/UWB-ACM/CTCI/tree/master/2019-03-Autumn/5_graphs#p3).

Reply:
Oops! Forgot to check past Winter 2020. Good to know for next week